packages/vscode-dependencies-validation/src/autofix/eventUtil.ts

```diff
@@ -10,6 +10,9 @@ import { VscodeFileEventConfig, VscodeWorkspace } from "../vscodeTypes";
 
 export const debouncedHandleProjectChange = debounce(handleProjectChange, 3000);
 
+// TODO: would this get invoked while package.json is being edited flow?
+//   may cause duplicate "npm i"...
```

Comment:
the cause of duplicate npm install message is not here i think

Reply:
Yeah, but I still want to debug this flow, it may be a bug/edge case